sdk/build.gradle

```diff
@@ -12,7 +12,7 @@ android {
     compileSdkVersion 28
     buildToolsVersion "28.0.3"
     defaultConfig {
-        minSdkVersion 14
+        minSdkVersion 23
```

Comment:
We could keep it as low as possible unless we really want to use something from newer APIs. We could set it to 19 so we also cover KitKats?

Reply:
I guess, there was something I wanted to use it for, I'll check what it was and get back to you.